src/pages/Bridge.vue

```diff
@@ -28,20 +34,21 @@
 import { mapGetters, mapActions } from "vuex";
 import metamask from "src/components/Metamask";
 import bridgeStepper from "../components/bridge/bridgeStepper.vue";
+import bridgeStepper2 from "../components/bridge/bridgeStepper2.vue";
 import bridgeDashboard from "src/components/bridge/TeleportDash.vue";
-import bridgeExternal from "src/components/bridge/ExternalBridge.vue";
 
 export default {
   components: {
     bridgeStepper,
+    bridgeStepper2,
```

Comment:
I assume bridge 1 is evm and stepper 2 is antelope? bit unclear

Reply:
Yes, that is true. It is used for the antelope bridge.